webcompat/templates/contributors/reproduce-bug.html

```diff
@@ -59,28 +59,28 @@ <h2 id="next" class="headline-1 highlight">Possible Next Steps</h2>
           </p>
 
           <dl id="statuses">
-            <dt id="incomplete" class="bold highlight">
+            <dt id="incomplete" class="bold headline">
```

Comment:
Ah, seeing how this is used elsewhere... can we use `heading` instead of `headline`?

Reply:
You mean just replace headline for heading in all of the places?